app/models/articles.model.js

```diff
@@ -18,10 +18,30 @@ const selectArticlesById = (articleId) => {
     return db.query(`SELECT * FROM articles WHERE article_id = $1`, [articleId])
         .then((result) => {
             if (result.rows.length === 0) {
-                return Promise.reject({status: 404, msg: "No article with specified ID found"})
+                return Promise.reject({status: 404, msg: "No article with specified ID found"});
             };
             return result.rows[0];
         });
 };
 
-module.exports = { selectArticlesById, selectArticles }
\ No newline at end of file
+const selectArticleComments = (articleId) => {
+    return db.query(`SELECT * FROM articles`)
+    .then((result) => {        
+        if (articleId > result.rows.length) {
```

Comment:
This is currently passing and I _think_ would work in all instances, but just to be safe I'd refactor it to state if `result.rows.length === 0` 😄